CHANGELOG.md

```diff
@@ -4,14 +4,26 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.38.0]
+
+### What's Changed
+
+* Provide better error message for invalid stubs by @JustusAdam in https://github.com/model-checking/kani/pull/2787
+* Deprecate any_slice by @zhassan-aws in https://github.com/model-checking/kani/pull/2789
```

Comment:
I think this one should be added under a "Major Changes" section.

Reply:
Done, please re-review.